### PR TITLE
Add caption to order description

### PR DESCRIPTION
**Pull request by magnusjohn-bjss**

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Order/Views/OrderDescription/OrderDescription.cshtml [hunk 1]
```diff
@@ -10,10 +10,13 @@
...
             <nhs-page-title title="@ViewBag.Title"
-                            advice="Provide a brief description of your order to help identify it from any others you’ve worked on." />
+                            caption="@Model.Caption"
+                            advice="@Model.Advice" />
             <form method="post">
                 <input type="hidden" asp-for="@Model.BackLink" />
                 <input type="hidden" asp-for="@Model.BackLinkText" />
+                <input type="hidden" asp-for="@Model.Caption" />
+                <input type="hidden" asp-for="@Model.Advice" />
                 <input type="hidden" asp-for="@Model.OdsCode" />
```
Comment (leerogers-bjss) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Order/Views/OrderDescription/OrderDescription.cshtml:20]: this hidden input for OdsCodes can be removed

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Order/Models/OrderDescription/OrderDescriptionModel.cs [hunk 2]
```diff
@@ -4,18 +4,35 @@ namespace NHSD.GPIT.BuyingCatalogue.WebApp.Areas.Order.Models.OrderDescription
...
+            OdsCode = odsCode;
+            Caption = organisationName;
+            Advice = NewOrderAdviceText;
+        }
+
         public OrderDescriptionModel(string odsCode, EntityFramework.Ordering.Models.Order order)
         {
             Title = "Order description";
             OdsCode = odsCode;
+            Caption = order?.CallOffId.ToString();
```
Comment (leerogers-bjss) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Order/Models/OrderDescription/OrderDescriptionModel.cs:26]: the caption should be the order "Order" followed by the calloffId
e.g "Order C10001-01"

**Review by leerogers-bjss — CHANGES_REQUESTED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Order/Views/OrderDescription/OrderDescription.cshtml (hunk 1 above)
Comment (magnusjohn-bjss) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Order/Views/OrderDescription/OrderDescription.cshtml]: removed

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Order/Models/OrderDescription/OrderDescriptionModel.cs (hunk 2 above)
Comment (magnusjohn-bjss) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Order/Models/OrderDescription/OrderDescriptionModel.cs]: updated

**Review by leerogers-bjss — APPROVED** [review]

**Review by leerogers-bjss — APPROVED** [review]

**Pull request merged**